cli/slsa-verifier/verify.go

```diff
@@ -62,6 +62,7 @@ func verifyArtifactCmd() *cobra.Command {
 
 			if _, err := v.Exec(cmd.Context(), args); err != nil {
 				fmt.Fprintf(os.Stderr, "%s: %v\n", FAILURE, err)
+				os.Exit(1)
```

Comment:
any chance we can have a pre-submit for it, or not worth it?
I was thinking of building the verifier and running it in a pre-submit. I'm not sure if it's worth it, since the code rarely changes and would be caught via the daily e2e runs.

Wdut?

Reply:
> any chance we can have a pre-submit for it, or not worth it?

Yeah, I think this is worth it, and I can write a follow-up using the example-package scripts. What would be nice is if we can re-use artifact/provenance from example-package though. So something like:
1. presubmit builds verifier at HEAD
2. fetches some latest artifacts from e2e package 
3. runs the same e2e.common.default.verify.sh script 

Or should we just check in some artifacts/use the testdata? Basically we just need to make sure the extensive e2e verifications are being run with the CLI itself.

Reply:
we can re-use all the test in main_test.go, on the CLI itself. I think that's what you're suggesting. Works for me!
https://github.com/slsa-framework/slsa-verifier/issues/382 is related, but I don't think we would catch the exit status in the code itself, ie we need to run the CLI in a script